instrument_edit.php

```diff
@@ -0,0 +1,459 @@
+      <td colspan="2" style="text-align:left;width:unset">
+       <input type="text" name="query_form_on[]" placeholder="condition logic" style="width:100%">
+      </td>
+     </tr>
+    </table>
+    <span id="inst-entries-link" style="display:none">
+     <a onclick="$('#inst-entries-tbl tr').slice(-2).clone(true).css('display',''
+                    ).insertBefore($('#inst-entries-tbl tr').slice(-2,-1));return false"
+        href="#" class=""><i class="fas fa-plus-circle fs12"></i> Add instrument</a>
+    </span>
+   </td>
+  </tr>
+  <tr>
+   <td>Condition</td>
+   <td>
+    <input type="text" name="query_where" style="width:100%" placeholder="condition logic"
+           value="<?php echo $module->escapeHTML( $reportData['where'] ?? '' ); ?>">
+   </td>
+  </tr>
+  <tr>
+   <td>Sorting</td>
+   <td>
+    <input type="text" name="query_orderby" style="width:100%" placeholder="sorting logic"
+           list="field-var-list-sort"
+           value="<?php echo $module->escapeHTML( $reportData['orderby'] ?? '' ); ?>">
+   </td>
+  </tr>
+  <tr>
+   <td>Fields to display</td>
+   <td style="padding:0px">
+    <table id="field-entries-tbl" style="width:95%;max-width:550px">
+     <tr>
+      <td style="text-align:left;width:60%">
+       <input type="text" name="query_select_field[]" placeholder="field name/logic"
+              value="<?php echo $module->escapeHTML( $reportData['select'][0]['field'] ?? '' ); ?>"
+              list="field-var-list" style="width:100%">
+      </td>
+      <td style="text-align:left;width:unset">
+       <input type="text" name="query_select_alias[]" placeholder="alias (optional)"
+              value="<?php echo $module->escapeHTML( $reportData['select'][0]['alias'] ?? '' ); ?>"
+              style="width:100%">
+      </td>
+     </tr>
+<?php
+$firstField = true;
+foreach ( $reportData['select'] as $fieldData )
+{
+	if ( $firstField )
+	{
+		$firstField = false;
+		continue;
+	}
+?>
+     <tr>
+      <td style="text-align:left;width:unset">
+       <input type="text" name="query_select_field[]" placeholder="field name/logic"
+              value="<?php echo $module->escapeHTML( $fieldData['field'] ); ?>"
+              list="field-var-list" style="width:100%">
+      </td>
+      <td style="text-align:left;width:unset">
+       <input type="text" name="query_select_alias[]" placeholder="alias (optional)"
+              value="<?php echo $module->escapeHTML( $fieldData['alias'] ); ?>"
+              style="width:100%">
+      </td>
+     </tr>
+<?php
+}
+?>
+     <tr style="display:none">
+      <td style="text-align:left;width:unset">
+       <input type="text" name="query_select_field[]" placeholder="field name/logic"
+              list="field-var-list" style="width:100%">
+      </td>
+      <td style="text-align:left;width:unset">
+       <input type="text" name="query_select_alias[]" placeholder="alias (optional)"
+              style="width:100%">
+      </td>
+     </tr>
```

Comment:
I am a bit confused here.  Can this <tr> remove?

Reply:
I meant above <tr> display:none section.

Reply:
These are the options which are added when 'add field' is clicked.

Reply:
If a field needs to be removed, the entry can be left blank.